pandas/core/series.py

```diff
@@ -1335,8 +1336,8 @@ def idxmax(self, axis=None, skipna=True, *args, **kwargs):
         return self.index[i]
 
     # ndarray compat
-    argmin = idxmin
-    argmax = idxmax
+    argmin = deprecate('argmin', idxmin)
+    argmax = deprecate('argmax', idxmax)
```

Comment:
I think we want a more custom deprecation message here. Using the `deprecate` decorator will just issue a "x is deprecated. Use y instead", while I think we want to mention eg that argmin/argmax will return the positional min/max in the future. 
So I would maybe just add an actual function definition of argmax and argmin looking something like this:

```
def argmin(self, axis=None, skipna=True, *args, **kwargs):
    """..."""
    warnings.warn(...)
    return self.idxmin(axis, skipna, *args, **kwargs)

```